Calculator.Tests/FunctionTests.cs

```diff
@@ -1,24 +1,110 @@
+namespace Calculator.Tests;
 using NUnit.Framework;
 using static Calculator.Functions;
 
-namespace Calculator.Tests;
-
 [TestFixture]
 public class FunctionTests
 {
     // good tests are split into the Three A's: Arrange, Act, and Assert
     [Test]
-    public void AddReturnsCorrectAnswer()
+    [TestCase(5, 4, 9)]    // Big positive number first, Small positive number second
+    [TestCase(4, 6, 10)]  //Small positive number, Big positive Number
+    [TestCase(-10, 6, -4)] //Big Negative Number, Small number
+    [TestCase(-12, 24, 12)] //Small Negative Number, Big Positive Number
+    [TestCase(-13, -7, -20)] //Big Negative Number, Small Negative Number
+    public void AddReturnsCorrectAnswer(int a, int b, int expectedSum)
     {
-        // arrange
+        // Arrange done by mapping TestCase Variables to Method Parameters
+        /*
         int a = 1;
         int b = 2;
         int expectedSum = 3;
+        */
 
         // act
-        int actualSum = Add(a, b);
+        var actualSum = Add(a, b);
 
         // assert
         Assert.AreEqual(expectedSum, actualSum);
     }
+
+    /*Fuction Name: SubtractReturnsCorrectAnswer
+    **Parameters: N/A
+    **Return: void
+    */
```

Comment:
I think we can omit these types of header comments for tests. For production non-test code C# has a format called XML comments. See [microsoft docs](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/).

I've definitely got into a bad habit of not using these when I should, many of the repos don't include XML comments. That said there are some newer projects where I use them (and even throw errors when not included on public members, classes, etc). Take a look at repo HarringtonProducts for an example of where I've been using them pretty much everywhere. I'm trying to make that repo a golden example of what we should be doing in terms of style and maintainability.